src/Plugin.php

```diff
@@ -141,18 +141,29 @@ private function loadInstalledPaths()
      */
     private function saveInstalledPaths()
     {
-        // By default we delete the installed paths
-        $arguments = ['--config-delete', 'installed_paths'];
+        $configKey = 'installed_paths';
```

Comment:
This `configKey` (`installed_paths`) is used multiple times in this class / Plugin (e.g. line 121).

I would really like to keep a single style. Right now it used as a variable here, while being used directly elsewhere. I would suggest on moving it into a class constant.